src/Header/Nav/Dropdown/Components/Components.module.scss

```diff
@@ -6,4 +6,8 @@
   margin-left: 0;
   max-width: none; // To override max-width of 66ch set on lists
   padding: 0;
+
+  li {
+    margin-bottom: rem($nds-spacing-small);
```

Comment:
Not sure what you're styling objective is but there are some list variants to choose from, see https://design-system.nice.org.uk/foundations/lists/

Reply:
Yeah on this one I went back and forth, because I'm having to bring the styling in directly to that sheet to override the list styles on all the services anyway. I had considered having it as a separate variant, but realised in this instance (ie in the dropdowns) I want all my lists the same anyway so it didn't seem sensible. Can split it back out though if you want so it is named as a separate variant.

Reply:
No that’s all good, makes sense 👍